ee/migrations/0007_migrate_definitions_tags.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.5 on 2022-01-28 19:21
+from django.db import migrations
+
+from posthog.models.tagged_item import EnterpriseTaggedItem
+
+
+def forwards(apps, schema_editor):
+    # Create new event definition tags
+    EnterpriseEventDefinition = apps.get_model("ee", "EnterpriseEventDefinition")
+    for instance in EnterpriseEventDefinition.objects.raw(
```

Comment:
Nit: Do we need to do a raw query here? I think we could just stick to the django orm?